packages/itwinui-react/src/core/Tree/Tree.tsx

```diff
@@ -393,13 +434,14 @@ const VirtualizedTree = React.forwardRef(
           className: cx(className, outerProps.className),
           style: { ...style, ...outerProps.style },
```

Comment:
Spread the `style` prop across the `outerProps` `style` in this [commit](https://github.com/iTwin/iTwinUI/pull/2091/commits/aba652b5ab824ccb3e51558f63e7de52ca61bbed). Also removed the `cx` call entirely, since there doesn't seem to be any `className` that should be combined with the given prop.

Reply:
Done in this [commit](https://github.com/iTwin/iTwinUI/pull/2091/commits/9515054618688ce46a75890518b516f3aa4f665c).